assets/css/main.scss

```diff
@@ -341,4 +341,9 @@ body.location-pasteur {
         margin: 0;
         font-size: 5em;
     }
-}
\ No newline at end of file
+}
+
+#rna-tooltable table td:nth-child(1), #rna-tooltable table td:nth-child(3){
```

Comment:
I would be fine to call this tooltable as we probably want to reuse it for other sides as well. @bebatut?